external-docs/docs/guides/configuration/convention.md

```diff
@@ -0,0 +1,37 @@
+# Configuration by convention
+
+This page details the approach that csvcubed takes to configuring cubes by convention. This makes it possible to generate a valid CSV-W cube without writing a [qube-config.json](./qube-config.md) at all. For an introduction to this topic, see the [quick start on designing your CSV](../../quick-start/designing-csv.md).
+
+## Minimum requirements
+
+The configuration by conventional approach in csvcubed requires that:
+
+* Your data set must be in the [standard data shape](../shape-data.md#standard-shape).
+* The data CSV's column titles use [conventional column names](#conventional-column-names) for [measure](../../glossary/index.md#measure) columns, [unit](../../glossary/index.md#unit) columns and [observed value](../../glossary/index.md#observation-observed-value) columns.
+
+Adhering to the [conventional column names](#conventional-column-names) is important since csvcubed uses those to understand what each column in your data set contains.
+
+## Inferred configuration
+
+In order to generate a cube, csvcubed needs some additional metadata and makes a few inferences to ensure it has all of the information it needs:
+
+* The title of the cube is the name of the csv file in [title case](https://en.wikipedia.org/wiki/Title_case) with underscores or dashes replaced by spaces.
+* Every column which does not use a conventional name is interpreted as a [dimension](../../glossary/index.md#dimension).
```

Comment:
"conventional name" 👉 "reserved name" ❓

Reply:
Reserved name has a different meaning elsewhere. We reserve the identifier 'code-list' in code lists. I don't want to confuse matters by calling different concepts the same name.